OpenRA.Mods.Common/Scripting/Properties/DockClientManagerProperties.cs

```diff
@@ -0,0 +1,88 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using Eluant;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("Ability")]
+	public class DockClientManagerProperties : ScriptActorProperties, Requires<DockClientBaseInfo>
```

Comment:
Shouldn't this require `DockClientManager` instead?

Reply:
sure